features/cart/viewing_cart/removing_cart_item.feature

```diff
@@ -0,0 +1,36 @@
+@viewing_cart
+Feature: Removing cart item from cart
+    In order to delete some unnecessary cart items
+    As a visitor
+    I want to be able to remove cart item
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "T-shirt banana" priced at "€12.54"
+        And the store has a product "T-shirt potato" priced at "€18.30"
+        And the cart has "T-shirt banana"
+
+    @todo
+    Scenario: Removing cart item
+        Given I open cart summary page
+        When I delete this product
+        Then I should be notified that this product has been successfully deleted
+        And I should be notified that my cart is empty
+
+    @todo
+    Scenario: Removing cart item from cart with multiple items
+        Given the cart also has "T-shirt potato"
+        And I open cart summary page
+        When I delete this product
+        Then I should be notified that this product has been successfully deleted
+        And unit price value should be "€12.54"
+        And I should see "Total"
```

Comment:
Do we need all those `And I should see "X" And X should be Y`? Isn't `And X should be Y` enough?